### DISCUSSION TITLE
Support for keyboard operations

### DISCUSSION THREAD

**Discussion by makotot** [TanStack/query#5571] [Ideas] [answered]
Is it out of the scope of the package's responsibility to add keyboard operation support in `@tanstack/react-query-devtools`?
I sometimes think it would be nice to have keyboard control for opening/closing processes, etc.

**Comment by TkDodo** [comment, marked as answer]
the button to open/close the devtools are interactive elements, so you can tab to them and open them. I wouldn't want the devtools to add global key event handlers in user's apps ...
